_posts/2016/2016-03-01-hr-java.md

```diff
@@ -0,0 +1,98 @@
+But it was also about non-development related stuff like ePages academy, plans of product management, scrum values etc.
+Besides that, I got to know other ePages newbies there.
+
+### And your team?
+
+Well, besides the training modules, I had short slots with the team.
+I was shown, what they're currently working on, how the sprint is going - and of course we had lunch together.
+
+### What happened after that week?
+
+I just jumped into the cold water.
+Originally, I was hired to start developing a new software architecture together with other colleagues.
+But first I supported the development of the ePages App & Theme Store.
+I also visited the Jena office to build a bridge to the colleagues I would be closely working with in the future.
+Communication is way easier, when you've already spent some time together.
+
+### The App & Theme Store is launched. What are you focussing on now?
+
+To being able to completely focus on the new architecture, we had to decouple this topic from the other tasks in the team.
+We split the team into two.
+Team Black stepped deep into the new architecture whereas the other colleagues focus on developing our REST API.
+
+### What does a typical work day look like?
+
+{% image blog/blog-team-black-2.jpg %}{% endimage %}
+
+I'm usually the early-bird in the team: I start between 8:00 and 8:30.
+Then I work myself through some smaller tasks, organisational stuff or do some bug fixing until we have our Daily Standup at 9:45.
+After having discussed the tasks for the day, we start single or pair programming sessions or discuss in detail about architectural concepts.
+
+Between 12:00 and 13:00 we all have lunch together.
+It's good to go somewhere outside, to have the head cleared.
+
+The afternoon is similar to the morning.
+Often we do sessions on the projector then, to discuss coding results with the whole team.
+Or I just put my headphones on and code away until the evening.
+
+### Which technologies do you use?
+
+We mainly work with Spring Boot, Docker, Kubernetes, RabbitMQ and MySQL.
+
+### What do you like at epages?
+
+The colleagues are all very open-minded and there's a friendly family atmosphere in the whole company.
+Between all teams there's an open interest for knowledge transfer.
+We also stepped deeper into the developer community and open more to the external development world.
+
+It's nice that ePages has flat hierarchies and clearly defined chains of command.
+
+I also like the cool company events, like the YOU or internal hackathons.
+
+### What qualities does an applicant need to fit into the team?
+
+Those who want to join Team Black should have children (*Jens is grinning*).
+Just kidding.
+But honestly, our team is above average with regards to children :wink:.
+
+They should not be afraid of criticism.
+We have intensive and professional discussions in the team, so they should be tough and convinced of their opinion.
```

Comment:
We often have intensive but professional discussion in the team, so they should be communicative and convinced of their opinion.